presto-product-tests/README.md

```diff
@@ -143,6 +143,12 @@ presto-product-tests/bin/run_on_docker.sh <profile> -t com.facebook.presto.tests
 presto-product-tests/bin/run_on_docker.sh <profile> -t sql_tests.testcases.system.selectInformationSchemaTables
 ```
 
+Profile specific tests are tests which runs only for that particular profile. It confirms if the profile is same as what is claims.
```

Comment:
> I added Presto docs as a backup. In case, the link for Hive Connector changes.

Changes are if someone is ready about writing product tests for Presto, they already know where to find the main Presto docs. So unless you want to direct users to a specific section, I say we should get rid of that link.

> If user wants to have more information on Kerberos/Impersonation, how to configure, which properties to set, etc. The link to Hive connector docs will be useful for that.

This makes sense because the Hive connector docs will have details on the different types of Kerberos authentication mechanisms. I would re-phrase in the following way:

For more information on the various ways in which Presto can be configured to interact with Kerberized Hive and Hadoop, please refer to [the Hive connector documentation](https://prestodb.io/docs/current/connector/hive.html).